docs/concepts.md

```diff
@@ -20,7 +20,7 @@ Ark is ideal for the disaster recovery use case, as well as for snapshotting you
 This section gives a quick overview of the Ark operation types.
 
 ### 1. Backups
-The *backup* operation (1) uploads a tarball of copied Kubernetes resources into cloud object storage and (2) uses the cloud provider API to make disk snapshots of persistent volumes, if specified. [Annotations][8] are cleared for PVs but kept for all other object types.
```

Comment:
It doesn't look like we reference url [8] anywhere any more. Remove it?

Reply:
done.

Reply:
Oh, I didn't mean you needed to renumber, but ok